src/project/internal/projectfilescontroller.cpp

```diff
@@ -486,9 +486,11 @@ io::path ProjectFilesController::selectScoreOpeningFile()
            << QObject::tr("MuseScore Dev Files") + " (*.mscs)"
            << QObject::tr("MuseScore Backup Files") + " (*.mscz~)";
 
-    return interactive()->selectOpeningFile(qtrc("project", "Score"), "", filter.join(";;"));
+    return interactive()->selectOpeningFile(qtrc("project", "Score"), configuration()->userProjectsPath() , filter.join(";;"));
 }
 
+
+
```

Comment:
Almost there! Now, please remove these two unnecessary newlines (we always put exactly one newline between two functions).

Then, please squash the commits again into one, and then I think this is ready to be tested and merged!

(Sorry for all these comments 😅)